docker/start.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+    echo "Setting the database"
+    ./diesel database setup
```

Comment:
would be nice to have commands to rollback and reset db also

Reply:
[start container script](https://github.com/pxp9/weather_bot_rust/pull/21/commits/b1969f365d59808d7e9f1391864349ade9c6fa03)